reana_client/cli/workflow.py

```diff
@@ -225,13 +235,15 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
     '--access-token',
     default=os.environ.get('REANA_ACCESS_TOKEN', None),
     help='Access token of the current user.')
-@click.argument(
-    'parameters',
-    nargs=-1,
+@click.option(
+    '-p', '--parameter',
+    multiple=True,
+    help='Operatinal parameters. For now only CACHING=[true|false] is '
```

Comment:
`s/Operatinal/Operational`

Instead of "for now only" you could say something short like:

```
Optional operational parameters for the workflow execution. E.g. CACHING=false.
```